chpl/chpl-service/src/main/java/gov/healthit/chpl/permissions/domains/activity/GetUserMaintenanceMetadataActionPermissions.java

```diff
@@ -0,0 +1,24 @@
+package gov.healthit.chpl.permissions.domains.activity;
+
+import org.springframework.stereotype.Component;
+
+import gov.healthit.chpl.permissions.domains.ActionPermissions;
+
+@Component("activityGetUserMaintenanceMetadataActionPermissions")
+public class GetUserMaintenanceMetadataActionPermissions extends ActionPermissions {
+
+    @Override
+    public boolean hasAccess() {
+        if (getResourcePermissions().isUserRoleAdmin() || getResourcePermissions().isUserRoleAcbAdmin()) {
```

Comment:
Not sure these permissions are right... ONC and ADMIN should see all user activity metadata and ACB/ATL/CMS should only see user activity metadata for their own users according to that massive security spreadsheet from a while ago.